### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by thisisjasleen** [weirme/FCSN#6]
Could you please provide a pre-trained model for the same?

**Comment by thisisjasleen** [comment]
I'm trying to run the train.py file. However, I'm getting an error in this line:
 tqdm.write(f'\nEpoch {epoch_i}')
Kindly help and please reply asap!
Thanks in advance!

**Comment by SinDongHwan** [comment]
@thisisjasleen 
Hi, This is [pretraind model(checkpoint)](https://1drv.ms/u/s!Au4J1yd2Q9NvlXKY9oJsPvjAzoyq?e=KxJZHU)
 

Could you tell me error message?
I want to run your code.
Did you edit self.data_path in config.py?
if not, edit self.data_path='./fcsn_dataset.h5' -> './fcsn_TVSum.h5'

**Comment by thisisjasleen** [comment]
@SinDongHwan
Hi, thankyou for replying.
I did edit the self.data_path .
The dataset was downloaded under the name 'fcsn_TVSum.h5'  but i changed it to the one already mentioned in the code i.e. 'fcsn_dataset.h5' 
The error still exists. A syntax error.

**Comment by SinDongHwan** [comment]
@thisisjasleen 
If you're okay, Can i see your code??
please send God.DongHwan@gmail.com or upload your repository.

**Comment by thisisjasleen** [comment]
@SinDongHwan
I've used the same code as in this repo.
Nothing has been changed.

**Comment by thisisjasleen** [comment]
@SinDongHwan
Can you share your code with me? The complete code for the implementation of this paper.
Here is my email: jasleenrissam8@gmail.com

**Comment by SinDongHwan** [comment]
@thisisjasleen 
Okay^^
I sent codes.

**Comment by thisisjasleen** [comment]
@SinDongHwan
Is your code complete? Is there any additional thing that needs to be added or should i directly run it?

**Comment by thisisjasleen** [comment]
@SinDongHwan
Also, I'm unable to open your pkl files.

**Comment by SinDongHwan** [comment]
@thisisjasleen 

I just edited "self.data_path" config.py. and then execute.
`python train.py.`
i didn't face error.

if you face error, please error message, where line, python version.
I can help exactly, because i don't know your situation detail.

**Comment by thisisjasleen** [comment]
@SinDongHwan
Could the error be due to tqdm?
I tried to run train.py in the exact same manner without changing anything in any of the codes bt still faced an error.

**Comment by SinDongHwan** [comment]
@thisisjasleen 
I think so, if just i see your statement.
i don't know detail error. 
could you tell me your error message?

**Comment by thisisjasleen** [comment]
File train.py, Line 74
        tqdm.write(f'\nEpoch {epoch_i}') 
Syntax Error:invalid syntax

**Comment by SinDongHwan** [comment]
what is your python version??

**Comment by thisisjasleen** [comment]
@SinDongHwan
I've installed tqdm. Don't know why is this error still coming.

**Comment by thisisjasleen** [comment]
Python 3.5.2

**Comment by SinDongHwan** [comment]
Can you edit Line 74?
`tqdm.write('\nEpoch {}'.format(epoch_i))`
please tell me, whether error on not.
if not, edit `tqdm.write` code like this using` .format()`

**Comment by thisisjasleen** [comment]
Thanks. That issue seems resolved now.
However, now I'm encountering a syntax error in this line:
ckpt_path = self.config.save_dir + f'/epoch-{epoch_i}.pkl'

**Comment by SinDongHwan** [comment]
Good!!
I think that "f" is string format. 
this code seems to support python 3.6 over.
python3.6 don't face error.
Please edit `f'{epoch_i}'` like this `'{}'.format(epoch_i)`
i will edit code. and then pull requests.

**Comment by thisisjasleen** [comment]
Okay. Thankyou for the help. 
Also, have you sent me the codes? Are those codes complete or do i have to add things?

**Comment by SinDongHwan** [comment]
@thisisjasleen 
Yes. But just this repository codes. so you can't.
Do you want executable codes??
I think nothing for train and test. maybe... hah

**Comment by thisisjasleen** [comment]
@SinDongHwan
Yes, I need complete executable codes.Preferably the trained models which i can further tune as per my needs. Could you provide them?

**Comment by thisisjasleen** [comment]
ckpt_path = self.config.save_dir + f'/epoch-{epoch_i}.pkl'
I tried editing this line like the way you suggested. It's showing some error though.

**Comment by SinDongHwan** [comment]
I just have a model made by this repository.
you can get same model, if you just execute.
`f'/epoch-{epoch_i}.pkl' -> '/epoch-{}.pkl'.format(epoch_i)`
It's easy^^.

**Comment by thisisjasleen** [comment]
Can you send me your model? Is it the one you sent previously? I'm not able to open those pickle files.

**Comment by SinDongHwan** [comment]
You means that pickle files crushed?
or can't download?

**Comment by thisisjasleen** [comment]
I've downloaded them. They just aren't opening.

**Comment by thisisjasleen** [comment]
@SinDongHwan
I'm  having a little trouble visualizing what the output of this entire code is>? I mean, what is the end result? Are we getting the summarized versions of the videos or not?

**Comment by SinDongHwan** [comment]
@thisisjasleen 
I think it's not hard to visualize.
this result is sequence of {0,1}. 
if you see, 'score_dir/epoch-{}.json',
```
key is video number.
value is sequence of {0,1}. 1: keyframe, 0:not
```
you should post process.
features of dataset pick one frame per 15 frame.
and then this is used to train, evaluate, test.

to get summarize video, read #3 . so you can see evaluate code link.

**Comment by thisisjasleen** [comment]
Okay, I'll check the link. Btw, what does this pickle file contain?

**Comment by Ai-is-light** [comment]
> pretraind model(checkpoint)

would you mind sharing your pre-trained model again? After linking the url "Hi, This is pretraind model(checkpoint)" you supported, I will got the following information
此项可能已被删除或过期，或者你没有查看权限。有关详细信息，请与此项的所有者联系。

**Comment by SinDongHwan** [comment]
hi, @Ai-is-light.
I can't access my server pc. 
I will re-upload a pretrained model on next Monday(19th).

if somebody has a pretrained model, please share it.

**Comment by tchennwpu** [comment]
> hi, @Ai-is-light.
> I can't access my server pc.
> I will re-upload a pretrained model on next Monday(19th).
> 
> if somebody has a pretrained model, please share it.

hi, could you please reshare the pre-trained model again?
